gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -858,8 +855,7 @@ gst_tensor_trainer_create_output_tensors (GstTensorTrainer * trainer,
     GstMemory ** out_mem, GstMapInfo * out_info)
```

Comment:
In my personal opinion, since we are not creating output tensors inside the function, shouldn't we also change the function name?

Reply:
I changed function name and update code to parse input buffer. plz review the changes again.